packages/firestore/src/platform_node/grpc_connection.ts

```diff
@@ -25,8 +25,7 @@ import {
 import firebase from '@firebase/app';
 const SDK_VERSION = firebase.SDK_VERSION;
 
-// eslint-disable-next-line @typescript-eslint/no-require-imports
-const grpcVersion = require('@grpc/grpc-js/package.json').version;
+const grpcVersion = import('@grpc/grpc-js/package.json').version;
```

Comment:
You could also do a normal import, then add a plugin in the [rollup.config.js](https://github.com/firebase/firebase-js-sdk/blob/master/packages/app/rollup.config.js#L94).
```
import { version } from '@grpc/grpc-js/package.json'
```

Reply:
We already use this build setting, but since we don't use `rollup` for our unit tests, this breaks the unit test.